frontend/src/game/game.tsx

```diff
@@ -26,6 +26,7 @@ export const Controls = {
 
 const Game = () => {
   const setGameState = useGameStore((state) => state.setGameState);
+  const [crosshairSelected, setCrosshairSelected] = useState(false);
```

Comment:
You probably don't need this. I think you can just use
```suggestion
  const [crosshairSelected, setCrosshairSelected] = useState(false);
  const activeSelection = useGameStore((s) => s.uiState.activeSelection);
  const crosshairSelected = activeSelection !== "none"
```